### PR TITLE
Tutors dashboard

### PR DESCRIPTION
**Pull request by ghost**
Screenshot_20200630-192215
Screenshot_20200630-192229
Screenshot_20200630-192251
Screenshot_20200630-192328

### CODE REVIEW

File: app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/AddBankFragment.kt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        }
+        val acc_name = view.findViewById<EditText>(R.id.acc_name)
+        val txt_acc_name = acc_name.text
+        val acc_number = view.findViewById<EditText>(R.id.acc_number)
+        val txt_acc_number = acc_number.text
+        val acc_route = view.findViewById<EditText>(R.id.acc_route)
+        val txt_acc_route = acc_route.text
+        val ss_number = view.findViewById<EditText>(R.id.acc_security_number)
+        val txt_ss_number = ss_number.text
+        val btn_save = view.findViewById<Button>(R.id.save)
```
Comment (mayokunadeniyi) [inline app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/AddBankFragment.kt]: Please make use of `ViewBinding` for your views. It is enabled in the project.

File: app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/CardDetails.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        super.onViewCreated(view, savedInstanceState)
+        navController = Navigation.findNavController(view)
+        val toolbar = view.findViewById<androidx.appcompat.widget.Toolbar>(R.id.toolbar)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            toolbar.setNavigationIcon(R.drawable.back_arrow)
+        }
+        val username = view.findViewById<TextView>(R.id.username)
+        val user_image = view.findViewById<CircleImageView>(R.id.user_image)
+        val btn_cancel =view.findViewById<Button>(R.id.btn_cancel)
+        val add_card = view.findViewById<LinearLayout>(R.id.add_card)
```
Comment (mayokunadeniyi) [inline app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/CardDetails.kt]: Make use of `ViewBinding`

File: app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/AddCardDetails.kt [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        val txt_accname = acc_name.text
+        val acc_number = view.findViewById<EditText>(R.id.acc_number)
+        val txt_accnumber = acc_number.text
+        val card_expiry_date = view.findViewById<EditText>(R.id.m_y)
+        val txt_card_expiry_date = card_expiry_date.text
+        val cw_number = view.findViewById<EditText>(R.id.cw)
+        val txt_cw_number = cw_number.text
+        val btn_save = view.findViewById<Button>(R.id.save)
+
+        btn_save.setOnClickListener {
```
Comment (mayokunadeniyi) [inline app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/AddCardDetails.kt]: Make use of `ViewBinding`.

File: app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/HomePageTutorFragment.kt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        super.onViewCreated(view, savedInstanceState)
+        navController = Navigation.findNavController(view)
+        val btn_sign_out = view.findViewById<Button>(R.id.signout)
+        val username = view.findViewById<TextView>(R.id.username)
+        var calendar = view.findViewById<CalendarView>(R.id.calendar)
+        val modify_btn = view.findViewById<Button>(R.id.modify_btn)
+        val card_detail = view.findViewById<LinearLayout>(R.id.card_details)
+        val withdrawal = view.findViewById<LinearLayout>(R.id.withdrawal)
+        val myBanks = view.findViewById<LinearLayout>(R.id.mybanks)
+        val tutor_classes_recycler = view.findViewById<RecyclerView>(R.id.tutor_classes)
```
Comment (mayokunadeniyi) [inline app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/HomePageTutorFragment.kt]: Make use of `ViewBinding`

File: app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/MakeWithdrawalFragment.kt [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+import androidx.navigation.Navigation
+import androidx.recyclerview.widget.RecyclerView
+import com.mobileforce.hometeach.R
+import de.hdodenhof.circleimageview.CircleImageView
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
```
Comment (mayokunadeniyi) [inline app/src/main/java/com/mobileforce/hometeach/TutorDashBoardFragments/MakeWithdrawalFragment.kt]: Remove any unused parameters.

**Review by mayokunadeniyi — CHANGES_REQUESTED** [review]
Hello there @mikailyusuf awesome stuff here! 👍  just a few changes and we should be good. Also, next time, before making a PR, in your own forked copy, ensure the project builds successfully, just click on the `Actions` to see the build process. When it is done and it passes, then make a PR. Also, the project makes use of `ViewBinding` please get rid of any unnecessary `findViewById` calls in this PR and make use of `ViewBinding` or `DataBinding` for your views. 

Lastly, this current build is failing:

Screenshot 2020-06-30 at 20 34 58

**Comment by mayokunadeniyi** [comment]
@mikailyusuf awesome stuff! This is good to get merged but there's 2 things you'll still have to get fixed. So sorry about this man.

1) The first screenshot, if you look at the 4 `CardView`s there, their edges appear to have this weird shadow, you'll either have to fix the margin around the `CardView`s or smth:
Screenshot 2020-07-01 at 03 00 48

2) The `CardView` in the second screenshot has the text too close to the top edge. Please provide some sort of spacing there.

Screenshot 2020-07-01 at 03 02 27

**Comment by ghost** [comment]
Screenshot from 2020-07-01 05-37-22
Screenshot from 2020-07-01 05-38-44
 Fixed Card View Corner

**Comment by mayokunadeniyi** [comment]
@mikailyusuf Awesome stuff man! Thank you 👍

**Pull request merged**